src/Database/Firebase.hs

```diff
@@ -32,22 +32,75 @@ query = Query Nothing Nothing Nothing Nothing
 key :: ToJSON a => a -> Key
 key = Key
 
-get :: (MonadIO m, MonadError e m, MonadReader r m, AsHttpError e, HasFirebase r, HasHttpCfg r, FromJSON a) => Location -> Maybe Query -> m a
+get :: (
+  MonadIO m, 
+  MonadError e m, 
```

Comment:
I don't have a strong opinion about listing the constraints on separate lines, but after seeing them side by side I think I prefer the one-line version. I think this preference comes from the feeling that putting each constraint on its own line de-emphasizes the types in favor of the constraints, but the constraints are less for me and more for the compiler. Again, I didn't even know I had an opinion on this until seeing them side by side. I will defer to @ralphmorton since he has more practical experience with these kinds of type annotations.